src/__init__.py

```diff
@@ -1,18 +1,43 @@
-from flask import Flask, request, jsonify
+from flask import Flask, request, jsonify, g
 from src.models.tagging_suggester import *
 import nltk
+from datetime import datetime
+import src.utils.app_config as app_config
+from models import Request
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
 
 app = Flask(__name__)
 
 tagging_suggester = TaggingSuggester()
 
+@app.before_request
+def before_request():
+    engine = create_engine(app_config.database_url(), strategy='threadlocal')
```

Comment:
Will this create a connection on every request? Ideally we'd store a connection that we can re-use...

Reply:
As discussed